be/src/io/cache_input_stream.cpp

```diff
@@ -47,6 +47,9 @@ CacheInputStream::CacheInputStream(std::shared_ptr<SeekableInputStream> stream,
 Status CacheInputStream::read_at_fully(int64_t offset, void* out, int64_t count) {
     BlockCache* cache = BlockCache::instance();
     count = std::min(_size - offset, count);
```

Comment:
What will happen is `count=0`?

Reply:
> What will happen is `count=0`?

If count=0, the upper layer also forward the offset, which lead the count less than zero, and return EndOfFile error.